api/src/Page/Shipment.php

```diff
@@ -3090,7 +3090,7 @@ function _cancel_pickup()
             $this->_error('No such lab contact');
         $cont = $cont[0];
 
-        $dewars = $this->db->pq("SELECT d.dewarid
+        $dewars = $this->db->pq("SELECT d.dewarid, d.storagelocation
```

Comment:
could the dewar storage location not be null?

Reply:
I've added a check for null on line 3116